.github/workflows/build.yml

```diff
@@ -35,6 +35,17 @@ jobs:
           subscriber:
             - 'litmus-portal/cluster-agents/subscriber/**'
 
+  goimports-checks:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.graphql-server == 'true' || needs.changes.outputs.authentication == 'true' || needs.changes.outputs.authentication == 'true' || needs.changes.outputs.event-tracker == 'true' || needs.changes.outputs.subscriber == 'true'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check Golang imports order
+      uses: Jerome1337/goimports-action@v1.0.3
+      with:
+        goimports-path: ./
+
   backend-checks:
```

Comment:
Maybe we can add the go imports step to the backend checks, I think that would make more sense

Reply:
Yes, it can be added as a step in the backend-checks job. A new job won't be required.